Clarification on sequenceId (#449)
VDA5050/VDA5050: VDA5050_EN.md

>Figure 6 Pseudocode of an order update. Note the change of the `orderUpdateId`.

This also aids in the event that an order update is lost (e.g., due to an unreliable wireless network).

> The AGV can always check that the last known base node has the same `nodeId` (and its `sequenceId`, more on that later) as the first new base node.

Reviewer: ```suggestion
The AGV can always check that the last known base node has the same `nodeId` (and `sequenceId`, more on that later) as the first node of a new order update.
```

This `sequenceId` runs over the nodes and edges (first node of an order receives a 0, the first edge then gets the 1, the second node then gets the 2, and so on).
This allows for easier tracking of the order progress.

Once a `sequenceId` is assigned and the node is released, it does not change with order updates (see Figure 7).

> This is necessary for both mobile robot and master control to determine which node of the order is refered to, in case a node is contained more than once (e.g., when driving a circular path).

Reviewer: ```suggestion
This is necessary to unambigiously refer nodes of the order, as nodes may be contained more than once (e.g., when driving a circular path).
```
Tried to simplify again
